flink-kubernetes-operator/pom.xml

```diff
@@ -195,32 +202,6 @@ under the License.
         </resources>
 
         <plugins>
-            <plugin>
```

Comment:
We should also move the `maven-antrun-plugin` usages for the backward compatibility check to the api plugin

Reply:
Done.